apps/dashboard/templates/meinberlin_dashboard/project_moderators.html

```diff
@@ -0,0 +1,47 @@
+{% extends "meinberlin_dashboard/base_dashboard.html" %}
+{% load i18n %}
+
+{% block dashboard_content %}
+    <h1>{{ project.name }}</h1>
+    <h2>{% trans 'Moderators' %}</h2>
+    <form method="POST">
+        {% csrf_token %}
+        <div class="form--single-line">
+            <div class="l-col-2">
+                {% include 'meinberlin_contrib/includes/form_field.html' with field=form.add_moderators %}
+            </div>
+            <div class="l-col-2">
+                <input type="submit" class="button button--secondary" value="{% trans 'Add' %}"/>
+            </div>
+        </div>
```

Comment:
- It does not feel very reusable to me.
- Wrapping the button in `.l-col-2` looks strange IMHO.
- At the very least, the `margin-top` should be removed for mobile.

This is really more of a feeling at this point. I do not really have a better idea.